templates/index.html

```diff
@@ -146,6 +146,12 @@ <h3><u>Pray For Remember Thee</u></h3>
           <p>Please keep Remember Thee and those working on it in your prayers. Our hope is that Remember Thee will draw you and those who use it into a closer relationship with God through His Son Jesus Christ, and support the work of God's kingdom here on earth.</p>
```

Comment:
We will also need to update this text.

"Please keep Remember Thee and those working on it in your prayers. Our hope is that it will draw you and those who use it into a closer relationship with God through His Son Jesus Christ. Let us know you're praying by pressing the 🙏 emoji below."

Reply:
bump on this @michaelyaacoub